src/store/getters.ts

```diff
@@ -68,7 +68,9 @@ export default {
     return state.currentFilingStatus
   },
 
-  // get last Change of Directors filing from list of past filings
+  /** Returns date of last Change of Directors filing from list of past filings. */
+  // FUTURE: this will break if we only retrieve a page of filings
+  // FUTURE: the API should be giving us this date
```

Comment:
Yes, I think the API should be giving the UI its "parameters". Two reasons:
1. there should be minimal business logic in the UI,
2. the API needs to calculate those parameters anyway to verify any data the UI posts to it.